Heakhafaw_1105.txt

```diff
@@ -0,0 +1,525 @@
+# Heakhafaw
+# -6,-4
+
+# Name: Heakhafaw (as)
+
+# Abbreviation: Heak
+
+# Milieu: M1105
+
+# Credits: Heakhafaw sector was designed by Peter G. Celella and James Holden and appears in The MegaTraveller Alien, Volume 2: Solomani &amp; Aslan (DGP, 1991). It has been further refined by Leigh Heron
+
+# Author:    Leigh Heron
+# Source:    Solomani and Aslan
+# Ref:       https://travellermap.com/data/heakhafaw
+
+# Subsector A: Staiahio
+# Subsector B: Aaotla
+# Subsector C: Tlah
+# Subsector D: Khte
+# Subsector E: Ihtofi
+# Subsector F: Kaeaa
+# Subsector G: Ileikhtaulr
+# Subsector H: Ihriyhiyr
+# Subsector I: Htyehrahao
+# Subsector J: Ileaseiha
+# Subsector K: Kyakiaha
+# Subsector L: Tleiftu'ea'
+# Subsector M: Kheseal
+# Subsector N: Eulrakhiyrl
+# Subsector O: Yooi
+# Subsector P: Eihoihkyowas
+
+# Alleg: AsMw: "Aslan Hierate, single multiple-world clan dominates"
+# Alleg: AsSc: "Aslan Hierate, multiple clans split control"
+# Alleg: AsT0: "Aslan Hierate, Tlaukhu control, Yerlyaruiwo (1), Hrawoao (13), Eisohiyw (14), Ferekhearl (19)"
+# Alleg: AsT1: "Aslan Hierate, Tlaukhu control, Khauleairl (2), Estoieie' (16), Toaseilwi (22)"
+# Alleg: AsT2: "Aslan Hierate, Tlaukhu control, Syoisuis (3)"
+# Alleg: AsT3: "Aslan Hierate, Tlaukhu control, Tralyeaeawi (4), Yulraleh (12), Aiheilar (25), Riyhalaei (28)"
+# Alleg: AsT4: "Aslan Hierate, Tlaukhu control, Eakhtiyho (5), Eteawyolei' (11), Fteweyeakh (23)"
+# Alleg: AsT5: "Aslan Hierate, Tlaukhu control, Hlyueawi (6), Isoitiyro (15)"
+# Alleg: AsT6: "Aslan Hierate, Tlaukhu control, Uiktawa (7), Iykyasea (17), Faowaou (27)"
+# Alleg: AsT7: "Aslan Hierate, Tlaukhu control, Ikhtealyo (8), Tlerfearlyo (20), Yehtahikh (24)"
+# Alleg: AsT8: "Aslan Hierate, Tlaukhu control, Seieakh (9), Akatoiloh (18), We'okunir (29)"
+# Alleg: AsT9: "Aslan Hierate, Tlaukhu control, Aokhalte (10), Sahao' (21), Ouokhoi (26)"
+# Alleg: AsTv: "Aslan Hierate, Tlaukhu vassal clan dominates"
+# Alleg: AsVc: "Aslan Hierate, vassal clan dominates"
+# Alleg: AsWc: "Aslan Hierate, single one-world clan dominates"
+# Alleg: AsXX: "Aslan Hierate, unclaimed"
+# Alleg: NaAs: "Non-Aligned, Aslan-dominated"
+# Alleg: NaHu: "Non-Aligned, Human-dominated"
+# Alleg: NaXX: "Non-Aligned, unclaimed"
+
+Hex  Name                      UWP       Remarks                    {Ix}   (Ex)    [Cx]   N   B  Z PBG W  A    Stellar       
+---- ------------------------- --------- -------------------------- ------ ------- ------ --- -- - --- -- ---- -----------------
+0208 Eailrayereal              E6368K6-A Ph                         {  0 } (773+1) [984C] -   R  - 502 10 AsMw M0 V

```

Comment:
Many of these values are not valid per T5 rules (e.g. the Gov and Law). e.g.: testng the first few lines with https://travellermap.com/tools/lintsec

Hint: Parsing as: T5 Second Survey - Column Delimited
Warning: UWP: Gov=19 out of range; should be: Pop(=8) + Flux, line 3: 0208 Eailrayereal              E6368K6-A Ph                         {  0 } (773+1) [984C] -   R  - 502 10 AsMw M0 V
Warning: UWP: Law=6 out of range; should be: Gov(=19) + Flux, line 3: 0208 Eailrayereal              E6368K6-A Ph                         {  0 } (773+1) [984C] -   R  - 502 10 AsMw M0 V
Warning: UWP: Gov=21 out of range; should be: Pop(=8) + Flux, line 4: 0401 Kiyoirai                  A9558M9-E Pa Ph                      {  2 } (678-3) [8A5C] -   R  - 221 10 AsVc G6 V
Warning: UWP: Law=9 out of range; should be: Gov(=21) + Flux, line 4: 0401 Kiyoirai                  A9558M9-E Pa Ph                      {  2 } (678-3) [8A5C] -   R  - 221 10 AsVc G6 V
Warning: UWP: Gov=20 out of range; should be: Pop(=6) + Flux, line 5: 0407 Ilaw                      C8A16L9-B Fl He Ni                   {  0 } (B55-5) [861C] -   T  - 232 14 AsT5 M1 V
Warning: UWP: Law=9 out of range; should be: Gov(=20) + Flux, line 5: 0407 Ilaw                      C8A16L9-B Fl He Ni                   {  0 } (B55-5) [861C] -   T  - 232 14 AsT5 M1 V
Warning: UWP: Gov=17 out of range; should be: Pop(=9) + Flux, line 7: 0610 Iiyhtakh                  C79A9HC-A Fr Hi In Pz Wa             {  3 } (D8C+2) [8C6C] -   R  A 231 10 AsSc K4 V M7 V
Warning: UWP: Gov=19 out of range; should be: Pop(=4) + Flux, line 8: 0706 Ieeiris                   C3364K8-A Ni                         {  0 } (A32+1) [246A] -   -  - 304 15 AsMw A7 V
Warning: UWP: Law=8 out of range; should be: Gov(=19) + Flux, line 8: 0706 Ieeiris                   C3364K8-A Ni                         {  0 } (A32+1) [246A] -   -  - 304 15 AsMw A7 V
Warning: UWP: Gov=21 out of range; should be: Pop(=8) + Flux, line 9: 0801 Auouearlei                C2228M9-C Na Ph Pi Po Sa             {  1 } (B77+5) [594C] -   -  - 331 11 AsVc M2 V
Warning: UWP: Law=9 out of range; should be: Gov(=21) + Flux, line 9: 0801 Auouearlei                C2228M9-C Na Ph Pi Po Sa             {  1 } (B77+5) [594C] -   -  - 331 11 AsVc M2 V
0 errors, 11 warnings.

Non-T5 data files can have... basically any data. But once T5 data files are used the rules should be followed where possible since we're trying to make a coherent and consistent data set.